org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.domainmodel/META-INF/MANIFEST.MF

```diff
@@ -1,34 +1,32 @@
-Manifest-Version: 1.0
-Bundle-ManifestVersion: 2
-Bundle-Name: Xtext Domainmodel Example
-Bundle-Vendor: Eclipse Xtext
-Bundle-Version: 2.10.0.qualifier
-Bundle-SymbolicName: org.eclipse.xtext.example.domainmodel; singleton:=true
-Bundle-ActivationPolicy: lazy
-Require-Bundle: org.eclipse.xtext;visibility:=reexport,
- org.eclipse.xtext.util,
- org.eclipse.emf.mwe2.launch;resolution:=optional,
- org.eclipse.emf.ecore;bundle-version="2.10.2",
- org.eclipse.emf.common;bundle-version="2.10.1",
- org.antlr.runtime;bundle-version="[3.2.0,3.2.1)",
- org.eclipse.xtext.xbase,
- org.eclipse.xtext.common.types,
- org.eclipse.xtext.xbase.lib;visibility:=reexport,
- org.eclipse.xtext.ui.codetemplates,
- org.objectweb.asm;bundle-version="[5.0.1,6.0.0)";resolution:=optional,
- org.eclipse.equinox.common,
- org.eclipse.xtend.lib
-Export-Package: org.eclipse.xtext.example.domainmodel,
- org.eclipse.xtext.example.domainmodel.domainmodel,
- org.eclipse.xtext.example.domainmodel.domainmodel.impl,
- org.eclipse.xtext.example.domainmodel.domainmodel.util,
- org.eclipse.xtext.example.domainmodel.formatting2,
- org.eclipse.xtext.example.domainmodel.jvmmodel,
- org.eclipse.xtext.example.domainmodel.parser.antlr,
- org.eclipse.xtext.example.domainmodel.parser.antlr.internal,
- org.eclipse.xtext.example.domainmodel.services,
- org.eclipse.xtext.example.domainmodel.validation,
- org.eclipse.xtext.example.domainmodel.serializer,
- org.eclipse.xtext.example.domainmodel.scoping
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Import-Package: org.apache.log4j;version="1.2.15"
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Xtext Domainmodel Example
+Bundle-Vendor: Eclipse Xtext
```

Comment:
As Bundle-Name and Vendor are directly in MANIFEST we can delete the plugin.properties file. Otherwise it is confusing.  
We should also setup the same line-wrap in xtext ws it is hard to review.